tests/handling/conftest.py

```diff
@@ -0,0 +1,189 @@
+"""
+Testing the handling of events on the top level.
+
+As the input:
+
+* Mocked cause detection, with the cause artificially simulated for each test.
+  The proper cause detection is tested elsewhere (see ``test_detection.py``).
+* Registered handlers in a global registry. Each handler is a normal function,
+  which calls a mock -- to ease the assertions.
+
+As the output, we check the mocked calls on the following:
```

Comment:
Output:

Check mock calls of: ...

Reply:
Not sure here about "check mock calls". Fixed to "check mockED calls". Why is it "mock calls"?

Reply:
I edited :)

Reply:
Actually I'd have made it less verbose, for instance
`*Output*
Check mocked calls of:
- bla
- bla `

Reply:
mkay, markdown is not helping

Reply:
Otherwise yes it's ok to have the "the" in there :)